addons/udes_stock/models/stock_picking.py

```diff
@@ -2784,130 +2784,137 @@ def raise_insufficiency_error(moves):
             to_reserve = picking_type.u_num_reservable_pickings
             reserve_all = to_reserve == -1
             base_domain = [("picking_type_id", "=", picking_type.id), ("state", "=", "confirmed")]
-            limit = 1
+            limit = 100 if reserve_all else picking_type.u_num_reservable_pickings
```

Comment:
100 stands for ?

Reply:
We're not setting `reserve_all` to -1, you missed an `=` I think.

100 is the number of pickings we will query for if `to_reserve` is -1, which means we must reserve as many pickings as possible.

Reply:
Missed that true

Reply:
Why exactly 100 :)

Reply:
> Why exactly 100 :)

It was @ws-mps's suggestion.